### PR TITLE
Home page logos

### PR DESCRIPTION
**Pull request by noobjey**
Added graphics to home page
Fixed issue with sign up page being hidden under header
Updated header to use icon and made menu items easier to see
removed unneeded js and css includes

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+  </div>
+</div>
+
+<div class="row">
+  <div class=col-lg-12">
+    <div class="login-form">
+      <%= form_for(:session, url: login_path) do |f| %>
+        <%= f.label :username %>
+        <%= f.text_field :username %>
+        <br/>
```
Comment (mrjaimisra) [inline app/views/sessions/new.html.erb]: remove breaks in the form

File: app/views/site/index.html.erb [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+    <!--cooked meal on the table in less than 15 minutes with no mess to clean up. It's that simple.</p>-->
 
-  <p>
-    <!--<a class="btn btn-primary btn-large">Call to action!</a>-->
-    <%= button_to "Lets Find A Meal", menu_path, html_options = { class: "btn btn-primary btn-large" } %>
-  </p>
+  <!--<p>-->
+    <!--&lt;!&ndash;<a class="btn btn-primary btn-large">Call to action!</a>&ndash;&gt;-->
+    <!--<%#= button_to "Lets Find A Meal", menu_path, html_options = { class: "btn btn-primary btn-large" } %>-->
+  <!--</p>-->
```
Comment (mrjaimisra) [inline app/views/site/index.html.erb]: what are all these comments?

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (noobjey) [inline app/views/sessions/new.html.erb]: will work on this when i figure out the correct way to do it.

**Comment by noobjey** [comment]
Cleaned up the commented out code. Will fix line breaks when i figure out a good way to do it.

**Pull request merged**